sysml/src/validation/05-State-based Behavior/5-State-based Behavior-1.sysml

```diff
@@ -82,7 +82,7 @@ package '5-State-based Behavior-1' {
 					first off
 					accept 'Vehicle Start Signal' 
 					if vehicle1_c1.'brake pedal depressed'
-					do send 'Start Signal'() to vehicle1_c1.vehicleController
+					do send new 'Start Signal'() to vehicle1_c1.vehicleController
```

Comment:
In line with the change, in the comment block below
`The notation "'Start Signal'()" constructs` should be changed to
`The notation "new 'Start Signal'()" constructs`

Reply:
Fixed.